Install mach_debug headers.  They seem to be required by <mach/gnumach.defs>.

--- Makefrag.am
+++ Makefrag.am
@@ -419,9 +419,16 @@ include_mach_eXec_HEADERS = \
 	include/mach/exec/elf.h \
 	include/mach/exec/exec.h
 
-# mach-debug-headers:= $(addprefix mach_debug/, hash_info.h ipc_info.h \
-# 		mach_debug.defs mach_debug_types.defs mach_debug_types.h \
-# 		pc_info.h vm_info.h slab_info.h)
+include_mach_debugdir = $(includedir)/mach_debug
+include_mach_debug_HEADERS := \
+	include/mach_debug/hash_info.h \
+	include/mach_debug/ipc_info.h \
+	include/mach_debug/mach_debug.defs \
+	include/mach_debug/mach_debug_types.defs \
+	include/mach_debug/mach_debug_types.h \
+	include/mach_debug/pc_info.h \
+	include/mach_debug/vm_info.h \
+	include/mach_debug/slab_info.h
 
 # Other headers for the distribution.  We don't install these, because the
 # GNU C library has correct versions for users to use.
